databases/mysql81/Portfile

```
@@ -1,373 +1,17 @@
-        reinplace "s|@SUBPORT@|${subport}|g" \
-            ${workpath}/org.macports.${subport}.plist
-        reinplace "s|@USER@|${mysqluser}|g" \
-            ${workpath}/org.macports.${subport}.plist
-        reinplace "s|@GROUP@|${mysqluser}|g" \
-            ${workpath}/org.macports.${subport}.plist
-        reinplace "s|@LOGFILE@|${log_file}|g" \
-            ${workpath}/org.macports.${subport}.plist
-    }
-
-    use_configure       no
-
-    build {}
-
-    destroot {
-        xinstall -d -m 0755 -o ${mysqluser} -g ${mysqluser} \
-            [file dirname ${destroot}${log_file}]
-
-        touch ${destroot}${log_file}
-        file attributes ${destroot}${log_file} -o ${mysqluser} -g ${mysqluser}
-
-        xinstall -d -m 0755 ${destroot}${prefix}/etc/LaunchDaemons/org.macports.${subport}
-
-        xinstall -m 0644 -o root -W ${workpath} \
-            org.macports.${subport}.plist \
-            ${destroot}${prefix}/etc/LaunchDaemons/org.macports.${subport}
-
-        xinstall -d -m 0755 ${destroot}/Library/LaunchDaemons
-
-        ln -s ${prefix}/etc/LaunchDaemons/org.macports.${subport}/org.macports.${subport}.plist \
-            ${destroot}/Library/LaunchDaemons/org.macports.${subport}.plist
-
-        xinstall -m 0755 -o root -d \
-            ${destroot}${prefix}/var/run
-
-        xinstall -m 0755 -o ${mysqluser} -g ${mysqluser} -d \
-            ${destroot}${prefix}/etc/${name_mysql} \
-            ${destroot}${prefix}/var/db/${name_mysql} \
-            ${destroot}${prefix}/var/log/${name_mysql} \
-            ${destroot}${prefix}/var/run/${name_mysql}
-
-        xinstall -m 0700 -o ${mysqluser} -g ${mysqluser} -d \
-            ${destroot}${prefix}/var/db/${name_mysql}-files \
-            ${destroot}${prefix}/var/db/${name_mysql}-keyring
-
-        destroot.keepdirs-append  \
-            ${destroot}${prefix}/var/db/${name_mysql} \
-            ${destroot}${prefix}/var/db/${name_mysql}-files \
-            ${destroot}${prefix}/var/db/${name_mysql}-keyring \
-            ${destroot}${prefix}/var/log/${name_mysql} \
-            ${destroot}${prefix}/var/run/${name_mysql}
-    }
-
-    notes "
-If this is a new install you might want to run:
-
-\$ sudo ${prefix}/lib/${name_mysql}/bin/mysqld --initialize --user=${mysqluser}
-\$ sudo port load ${name_mysql}-server
-\$ ${prefix}/lib/${name_mysql}/bin/mysql_secure_installation
-
-The first command creates the necessary files for the MySQL database service.
-(Remember to make a note of the auto-generated root password from this step.)
-The second command starts the MySQL service.
-The last command helps to improve the security of your running MySQL instance.
-
-Once enabled, the MySQL logs can be found in:
-[file dirname $log_file]
-"
-
-    livecheck.type          none
+PortSystem          1.0
+PortGroup           obsolete 1.0
+
+name                mysql81
+replaced_by         mysql8
+version             8.1.0
+revision            5
```

The revision must be higher than the previous revision for the upgrade to be offered to the user.
```suggestion
revision            6
```